src/main/resources/build.properties

```diff
@@ -6,3 +6,6 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Previous=Create property
+Next=Create property
+Page\:=Create property
```

Comment:
I don't understand this? -- Did you modify the wrong file? `JabRef_en.properties`. See https://devdocs.jabref.org/code-howtos/localization.html for more information

Reply:
When I create a button name and page layout it will show error if I didn't do that.